meta-fobnail-distro/recipes-fobnail/libcoap/libcoap_%.bb

```diff
@@ -0,0 +1,37 @@
+LICENSE = "Unknown"
```

Comment:
@kotylamichal please lets add here description and homepage to follow some structure of recipes https://docs.yoctoproject.org/dev-manual/common-tasks.html#locating-and-using-a-similar-recipe

Reply:
Also, can't we get LICENSE from https://github.com/obgm/libcoap#license-information ?

Reply:
Description and homepage added, license too - it is `BSD 2-Clause “Simplified” License`